archlinuxcn/smiley-sans-git/lilac.yaml

```
@@ -0,0 +1,14 @@
+maintainers:
+  - github: taotieren
+
+build_prefix: extra-x86_64
```

```suggestion
build_prefix: archlinuxcn-x86_64
```

在 `archlinuxcn` 源加入 `python-fontmake` 后，需要通过 `archlinuxcn-x86_64` 来安装包含于 `archlinuxcn` 源中的依赖。

---

已更新。谢谢。